app/views/secondary_content_links/index.html.erb

```diff
@@ -41,9 +41,6 @@
             } %>
 
             <%= render "govuk_publishing_components/components/input", {
```

Comment:
<%= render "govuk_publishing_components/components/input", {
  label: {
    text: "What is your name?"
  },
  name: "name",
  heading_size: "l"
} %>
```